src/ncp/ncp_base_mtd.cpp

```diff
@@ -2617,6 +2619,32 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_THREAD_NETWORK_TIME>(
 exit:
     return error;
 }
+
+void NcpBase::HandleTimeSyncUpdate(void *aContext, int64_t aNetworkTimeOffsetDelta)
+{
+    static_cast<NcpBase *>(aContext)->HandleTimeSyncUpdate(aNetworkTimeOffsetDelta);
+}
+
+void NcpBase::HandleTimeSyncUpdate(int64_t aNetworkTimeOffsetDelta)
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(ABS(aNetworkTimeOffsetDelta) >= OPENTHREAD_CONFIG_NCP_TIME_SYNC_JUMP_NOTIF_MIN_US);
+
+    SuccessOrExit(error = mEncoder.BeginFrame(SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0, SPINEL_CMD_PROP_VALUE_IS,
```

Comment:
Thanks for submitting this PR.
Suggestion about the `ncp_base` changes: How about adding `SPINEL_PROP_THREAD_NETWORK_TIME` to `ChangedPropsSet::mSupportedProps` list and then from here we can do this:
```
VerifyOrExit(ABS(aNetworkTimeOffsetDelta) >= OPENTHREAD_CONFIG_NCP_TIME_SYNC_JUMP_NOTIF_MIN_US);
mChangedPropsSet.AddProperty(SPINEL_PROP_THREAD_NETWORK_TIME);
mUpdateChangedPropsTask.Post();
```

Reply:
Thanks for the suggestion, looks a lot neater. Changed.